rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/is_hvac_sys_cooling_type_DX.py

```diff
@@ -0,0 +1,34 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.utils.jsonpath_utils import find_exactly_one_with_field_value, find_one
+
+COOLING_SYSTEM_TYPE = schema_enums["CoolingSystemOptions"]
+
+
+def is_hvac_sys_cooling_type_dx(rmi_b, hvac_b_id):
+    """Returns TRUE if the HVAC system has DX cooling. Returns FALSE if the HVAC system has anything other than DX cooling or if it has more than 1 or no cooling system.
```

Comment:
The second sentence should be updated to match the updated description in the RDS.

Reply:
Thanks for catching this. I just merged the updated RDS this morning and havent got time to review the open PRs. Will do to update this function